selfdrive/car/mazda/values.py

```diff
@@ -8,13 +8,13 @@
 # Steer torque limits
 
 class CarControllerParams:
-  STEER_MAX = 600                 # max_steer 2048
-  STEER_STEP = 1                  # how often we update the steer cmd
+  STEER_MAX = 1500                # theoretical max_steer 2047
```

Comment:
why was this raised? is this valid for all cars?

Reply:
Hi @adeebshihadeh, instead of pumping this to 1500, I will collect some data on the behavior at different values and select the max value that works best. I will update this PR when I get a chance.

Reply:
The EPS in older cars max at 308, the new ones at 620. In my testing with older cars I selected 600 based on testing. Even though the EPS will trim values above 308, it responds better to higher values by ramping up the torque faster. Reviewing drive data from newer cars show that the don't have the ramp up delay we had in older cars. so a value in the range 620-800 would work best across the board.